win32ss/user/ntuser/display.c

```diff
@@ -280,9 +281,53 @@ UserEnumDisplayDevices(
     RtlStringCbCopyW(pdispdev->DeviceName, sizeof(pdispdev->DeviceName), pGraphicsDevice->szWinDeviceName);
     RtlStringCbCopyW(pdispdev->DeviceString, sizeof(pdispdev->DeviceString), pGraphicsDevice->pwszDescription);
     pdispdev->StateFlags = pGraphicsDevice->StateFlags;
-    // FIXME: fill in DEVICE ID
     pdispdev->DeviceID[0] = UNICODE_NULL;
 
+    if (pGraphicsDevice->PhysDeviceHandle != NULL)
```

Comment:
Maybe add a comment explaining what is going to be done there, and the reason why only the first hardware ID is the one considered?

Reply:
Do these different IDs have different formats?

Reply:
![image](https://user-images.githubusercontent.com/578406/170087812-5e2a7a1e-253f-4190-91c8-30eaf5adf6c1.png)

Just a bit.

Reply:
I have added missing bits regarding the monitor handling. As it turned out, there is a lot of work needs to be done for it, and I don't want to interfere with Hervé's work on multiple monitors. :smile: